fda-model/metrics/ProjectPerformanceResults.cs

```diff
@@ -85,6 +85,7 @@ public double AssuranceOfAEP(double exceedanceProbability)
         public bool ConditionalNonExceedanceProbabilityIsConverged()
         {
             //dont like this.
+            //couldn't we nix the foreach and just return _cnep[0.98].IsConverged?
```

Comment:
yes, but it was not working for me. please test.
i think it was some issue regarding equivalency...

Reply:
Will do

Reply:
done